CONTRIBUTING.md

```diff
@@ -157,30 +157,93 @@ If you have made any significant visual changes to a component, we encourage you
 pnpm storybook
 ```
 
-## 🕸️ Gradio Website
+## ✍️ Gradio Website & Docs 
 
-We also welcome any contributions to our [website](https://www.gradio.app). 
+We also welcome any contributions to our [website](https://www.gradio.app) and [docs](https://www.gradio.app/docs). 
 
-First, build the website:
+### Building The Website
+
+All of the website code lives in the `js/_website/` directory. 
+
+To start the website on dev mode simply cd into this directory and run: 
 
 ```
-pnpm build:cdn-local
+pnpm dev
```

Comment:
You may need to do `pnpm i` first, right?

Reply:
For me worked pnpm dev

Reply:
If it hasn't been run before then pnpm i will be required. Can be run from anywhere.